packages/koa-shopify-auth/src/auth/test/oauth-callback.test.ts

```diff
@@ -152,7 +152,7 @@ describe('OAuthCallback', () => {
     const {secret, apiKey} = baseConfig;
     const {code} = queryData;
 
-    expect(fetchMock.lastCall()).toStrictEqual([
```

Comment:
This change was made after getting [an error (`Received: serializes to the same string`) running the tests](https://github.com/facebook/jest/issues/8475). The [resolution was documented in the Jest GitHub issue](https://github.com/facebook/jest/issues/8475#issuecomment-495729482).